### PR TITLE
Make the browser key site_config aware

### PR DESCRIPTION
**Pull request by Ivanca**
Use SEB_KEYS added to the Site Configuration object related to the correct site.

### CODE REVIEW

File: seb_openedx/seb_keys_sources.py [hunk 1]
```diff
@@ -32,5 +33,16 @@ def from_global_settings(course_key):
...
+    """
+    if configuration_helpers.has_override_value('SEB_KEYS'):
+        keys_dict = configuration_helpers.get_configuration_value('SEB_KEYS')
+        return keys_dict.get(six.text_type(course_key), None)
+    return None
+
+
 # First one has precedence over second, second over third and so forth.
-ORDERED_SEB_KEYS_SOURCES = [from_global_settings, from_other_course_settings]
+# ORDERED_SEB_KEYS_SOURCES = [from_global_settings, from_other_course_settings, from_site_configuration]
```
Comment (felipemontoya) [inline seb_openedx/seb_keys_sources.py]: Remove unused code

File: seb_openedx/seb_keys_sources.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 from django.utils import six
 from django.conf import settings
 from seb_openedx.edxapp_wrapper.get_course_module import get_course_module
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```
Comment (felipemontoya) [inline seb_openedx/seb_keys_sources.py]: You probably need to get this through a backend

**Review by felipemontoya — CHANGES_REQUESTED** [review]

**Comment by Ivanca** [comment]
@felipemontoya  Feedback addressed on last commit (ammended)

File: seb_openedx/seb_keys_sources.py [hunk 3]
```diff
@@ -32,5 +33,16 @@ def from_global_settings(course_key):
...
+    configuration_helpers = get_configuration_helpers()
+    if configuration_helpers.has_override_value('SEB_KEYS'):
+        keys_dict = configuration_helpers.get_configuration_value('SEB_KEYS')
+        return keys_dict.get(six.text_type(course_key), None)
+    return None
+
+
 # First one has precedence over second, second over third and so forth.
-ORDERED_SEB_KEYS_SOURCES = [from_global_settings, from_other_course_settings]
+ORDERED_SEB_KEYS_SOURCES = [from_global_settings, from_other_course_settings, from_site_configuration]
```
Comment (felipemontoya) [inline seb_openedx/seb_keys_sources.py:48]: Given that the precedence is higher in the first value, I think our current implementation should set the preference as:
1) from_other_course_settings
2) from_site_configuration
3) from_global_settings
What is your take on this? Also, perhaps this should go in a different PR once we get there

File: seb_openedx/settings/common.py [hunk 4]
```diff
@@ -29,3 +29,4 @@ def plugin_settings(settings):
     See: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/plugins/README.rst
     """
     settings.EOX_CORE_COURSE_MODULE = 'seb_openedx.edxapp_wrapper.backends.get_course_module_h_v1'
+    settings.EOX_CORE_CONFIGURATION_HELPERS = 'seb_openedx.edxapp_wrapper.backends.get_configuration_helpers_h_v1'
```
Comment (felipemontoya) [inline seb_openedx/settings/common.py]: The name should not be EOX_CORE for anything in this plugin

**Review by felipemontoya — CHANGES_REQUESTED** [review]

File: seb_openedx/seb_keys_sources.py (hunk 3 above)
Comment (Ivanca) [reply seb_openedx/seb_keys_sources.py:48]: I don't think there is a clear order for them; maybe the best would be to merge all the available keys (but for a different PR)

File: seb_openedx/settings/common.py (hunk 4 above)
Comment (Ivanca) [reply seb_openedx/settings/common.py]: Fixed (the one added in this PR, the other mention of EOX_CORE should be fixed in a new one I will make)

**Review by felipemontoya — APPROVED** [review]
